docs/developers/linea-version/release-notes.mdx

```diff
@@ -17,7 +17,52 @@ Find out all you need to know about the latest Linea versions.
 >
 > | Network | Version | Deployment Date |
 > | --- | --- | --- |
-> | Linea Goerli | [Alpha v0.2.0](#linea-alpha-v020-june-13-2023) | June 13, 2020 |
+> | Linea Goerli | [Alpha v0.2.3](#linea-alpha-v023-june-13-22-23) | June 13, 2020 |
+
+## Linea Alpha v0.2.3 (June 22, 2023)
+
+### Summary
+
+This release focuses on optimizing the provers hashing strategy to increase its computational efficiency.
+
+## Linea Alpha v0.2.2 (June 20, 2023)
+
+### Summary
+
+This release focuses on updating the Postman Sdk configuration values and revising gas fee calculations.
+
+### Features
+
+- Updated the Postman Sdk configuration files to handle zero values and revised gas fee calculations.
```

Comment:
I would be more explicit with something like 
```
to handle messages that are contract calls with zero ETH value being transferred
```

Reply:
fixed